src/cmap/auth/mongodb_oidc/token_entry_cache.ts

```diff
@@ -0,0 +1,110 @@
+import type { OIDCMechanismServerStep1, OIDCRequestTokenResult } from '../mongodb_oidc';
+
+/* 5 minutes in milliseonds */
+const EXPIRATION_BUFFER_MS = 300000;
+/* 5 hours in seconds */
+const DEFAULT_EXPIRATION_SECS = 0;
```

Comment:
the comment above this line says it's supposed to be 5 hours, and the comment in L99 says the same thing - is the default to expire immediately? what's the intended spec behavior?

Reply:
Updated the comment. The default in the spec is 0/expire immediately but I forgot to change the comment when I fixed it.